java/de/yessoft/android/mohammad/model/NewsItem.java

```diff
@@ -0,0 +1,87 @@
+package de.yessoft.android.mohammad.model;
+
+import android.util.ArrayMap;
+import android.util.Log;
+
+import com.google.firebase.firestore.DocumentSnapshot;
+
+import java.util.List;
+import java.util.Map;
+
+public class NewsItem {
+    private static final String TAG = "ML_NewsItem";
+
+    private static final String KEY_HEADING = "heading";
+    private static final String KEY_IMG = "img";
+    private static final String KEY_SUMMERY = "summery";
+    private static final String KEY_FULL_NEWS_ID = "full_news_id";
+
+    private String Heading;
```

Comment:
variable names should start with small letter

Reply:
as a general rule you would be absolutely right, however in my case, and Android as general, Old-Time Java artists would use mHeading, in my case i wanted to get the most benefits of the automation from Android studio, and to get the methods getHeading() generated, plus the variable is private so nothing is allowed in, however i can change the style of this rather easily